lib/proxy/log_buffer/ring_buffer.rb

```diff
@@ -0,0 +1,56 @@
+module Proxy::LogBuffer
```

Comment:
You should probably add attribution in every file that has celluloid code in it...